circle.yml

```diff
@@ -4,7 +4,7 @@ machine:
   python:
     version: 2.7.10
   node:
-    version: 0.12.0
+    version: 4.5.0
```

Comment:
@pmac is this ok to set as the Node version CircleCI uses? I'm not entirely sure what version we're using in dockerfiles, other than it being 4.x.